docs/DeveloperManuals/Project.md

```diff
@@ -0,0 +1,226 @@
+
+It contains the following two models:
+ - `projects` describes a project object, including its name, creation and update time and other basic information
+ - `project_metrics` describes the mapping relationship enabled by a plugin, including the name of the project and plugin on both sides of the mapping, and basic information about plugin options.
+
+![image](project_table.jpg)
+
+## projects
+
+|   **field**   | **type** | **length** | **description**               | **key** |
+| ------------- | -------- | ---------- | ----------------------------- | ------- |
+| `name`        | varchar  | 255        | name for project              | PK      |
+| `description` | longtext |            | description of the project    |         |
+| `created_at`  | datetime | 3          | created time of project       |         |
+| `updated_at`  | datetime | 3          | last updated time of project  |         | 
+
+
+| **name**  | **describe**                         | **created_at**          | **updated_at**          |
+| --------- | ------------------------------------ | ----------------------- | ------------------------|
+| project_1 | this is one of the test projects     | 2022-11-01 01:22:13.000 | 2022-11-01 02:24:15.000 |
+| project_2 | this is another project test project | 2022-11-01 01:23:29.000 | 2022-11-01 02:27:24.000 |
+
+## project_metrics
+
+|    **field**    | **type** | **length** | **description**                                            | **key** |
+| --------------- | -------- | ---------- | ---------------------------------------------------------- | ------- |
+| `project_name`  | varchar  | 255        | name for project                                           | PK      |
+| `plugin_name`   | varchar  | 255        | name for plugin                                            | PK      |
+| `plugin_option` | longtext |            | check if metric plugins have been enabled by the project   |         |
+| `enable`        | tinyint  | 1          | if the metric plugins is enabled                           |         |
+
+
+| **project_name** | **plugin_name** | **plugin_option** | **enable** |
+| ---------------- | --------------- | ----------------- | ---------- |
+| project_1        | gitlab          | {}                | true       |
+| project_2        | gitlab          | {}                | false      |
+| project_2        | github          | {}                | true       |
+
+## project_mapping
+
+|   **field**    | **type** | **length** | **description**                                            | **key** |
+| -------------- | -------- | ---------- | ---------------------------------------------------------- | ------- |
+| `project_name` | varchar  | 255        | name for project                                           | PK      |
+| `scope_id`     | bigint   | 255        | the scope id  by project mapping                           | PK      |
```

Comment:
it doesn't seem right, we had this table already, it should contains `project_name` `table` and `row_id`